src/MIT-open-group.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+    <license isOsiApproved="false" licenseId="MIT-open-group"
+             name="MIT Open Group variant">
+        <text>
```

Comment:
Before the `<text>` block, I would add a `<crossRefs>` list with at least one entry with a URL showing use of this license in the wild. Here is an example of what this can look like -- you can add this one as-is:

```
<crossRefs>
  <crossRef>https://gitlab.freedesktop.org/xorg/app/iceauth/-/blob/master/COPYING</crossRef>
</crossRefs>
```

Reply:
Upon searching for the entire license using some search engines, I couldn't find any other matches for the license which were exactly the same (except for the alt part of course).

Will it work if I don't add any other cross-references?